features/makefiles/helloapp/conanfile.py

```diff
@@ -0,0 +1,26 @@
+from conans import ConanFile, AutoToolsBuildEnvironment
+from conans import tools
+
+
+class AppConan(ConanFile):
+    name = "app"
+    version = "0.1"
+    settings = "os", "compiler", "build_type", "arch"
+    exports_sources = "src/*"
+    requires = "hello/0.1@user/testing"
+
+    def build_requirements(self):
+        if self.settings.os == "Windows":
+            self.build_requires("make/4.2.1")
```

Comment:
This was added to make the example build on windows but maybe is too complex for the example in the docs

Reply:
Should we skip this example on windows and remove this?

Reply:
Probably yes, this example can assume that you have Make in the system. Better to keep examples as lean as possible.

Reply:
But make sure to put a big comment with a warning about this.

Reply:
I think so. As examples are supposed to be used by newbies, I believe only Linux users will run make.